src/utils.js

```diff
@@ -242,9 +242,22 @@ function camelcase (dashCaseText) {
  * @param {string} prop
  * @returns {any}
  */
-export function getFeatureSetting (featureName, args, prop) {
+export function getFeatureSetting (featureName, args, prop, defaultValue) {
     const camelFeatureName = camelcase(featureName)
-    return args.featureSettings?.[camelFeatureName]?.[prop]
+    const configSetting = args.featureSettings?.[camelFeatureName]?.[prop]
+
+    // Config may pass the string 'undefined' which should be interpreted as undefined
+    // Otherwise return the config value
+    if (configSetting !== undefined) {
```

Comment:
Can we place this logic change in a function that wraps _getFeatureSetting_ aka _getAttributeValue_ and then express the setting value as an object in the config?

The reason being is you may in future wish to express other types not expressible in JSON like ArrayBuffer or NaN, Symbol or something like that.

In the config this could look something like:

```
"settings": {
    "keyboard": {type: 'undefined'},
    "availLeft": {type: 'number', value: 12},
 }
 ```
 

I don't think the rest of the code will want to support the 'undefined' string value.

Reply:
Fair enough 👍